src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java

```diff
@@ -134,4 +141,19 @@ private static void renderBox(final BlockPos posA, final BlockPos posB, final Cl
 
         RenderSystem.disableDepthTest();
     }
+
+    /**
+     * Used to catch the clientTickEvent.
+     * Call renderer cache cleaning every 5 secs (100 ticks).
+     *
+     * @param event the catched event.
+     */
+    @SubscribeEvent
+    public static void onClientTickEvent(final ClientTickEvent event)
+    {
+        if (Minecraft.getInstance().world.getGameTime() % (Constants.TICKS_SECOND * 5) == 0)
+        {
+            BlueprintHandler.getInstance().cleanCache();
```

Comment:
Nope! This is way to much of a CPU hog, use a Guava cache which can do this for you automatically!